tools/mirror/src/cli.rs

```diff
@@ -67,16 +61,17 @@ impl RunCmd {
             }
             None
         };
-
+        
```

Comment:
Trailing whitespace is unnecessary. Have you run `cargo fmt` after making your changes?